crmsh/sbd.py

```diff
@@ -45,26 +44,31 @@ def __init__(self, context):
         self._stonith_watchdog_timeout = self.STONITH_WATCHDOG_TIMEOUT_DEFAULT
         self._stonith_timeout = 60
         self._sbd_watchdog_timeout = 0
```

Comment:
Nice suggestions, thank you!

Reply:
I think I agree with Yan to keep some crmsh hard-coded defaults. Though, Xin tried to avoid the deplication between `crmsh hard-code` and `profiles.yml` ;)

Reply:
This is why I'd just drop s390 profile to prevent unnecessary duplication there, given that it's consistent with internal default, and we have to have the internal default anyway.

Profile is a place where users configure things that are different from the defaults rather than where users get to know the defaults.

Reply:
OK, dropped "s390" part